src/styles/page-template/index.md.njk

```diff
@@ -98,6 +98,13 @@ Variables can be set with:
       <td class="govuk-table__cell">Add a class to the <code>&lt;html&gt;</code> element.</td>
     </tr>
 
+    <tr class="govuk-table__row">
```

Comment:
May want considering adding these to the 'Customised page template' example

Reply:
Thanks Nick that was a good shout 👍